main.go

```diff
@@ -3,21 +3,29 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"io/ioutil"
 	"net"
 	"net/http"
 	"os"
+	"strings"
 
 	"github.com/pocke/hlog"
 	"github.com/skratchdot/open-golang/open"
 	"github.com/spf13/pflag"
 )
 
 func main() {
+	conf, err := loadConfigFile()
+	if err != nil {
+		panic(err)
+	}
+
 	var port int
 	var binding string
-	pflag.IntVarP(&port, "port", "p", 0, "TCP port number")
-	pflag.StringVarP(&binding, "binding", "b", "localhost", "Bind www to the specified IP.")
-	pflag.Parse()
+	fs := pflag.NewFlagSet(os.Args[0], pflag.ExitOnError)
+	fs.IntVarP(&port, "port", "p", 0, "TCP port number")
+	fs.StringVarP(&binding, "binding", "b", "localhost", "Bind www to the specified IP.")
+	fs.Parse(append(conf, os.Args...))
```

Comment:
__[gometalinter]__
* [errcheck] error return value not checked (fs.Parse(append(conf, os.Args...)))